app/src/main/res/layout-land/quest_answer_fragment.xml

```diff
@@ -0,0 +1,124 @@
+                        android:contentDescription="@string/close"
+                        android:adjustViewBounds="true"
+                        android:padding="12dp"/>
+
+                    <TextView
+                        android:id="@+id/title"
+                        style="@style/QuestTitleQuestion"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:paddingRight="16dp"
+                        android:paddingLeft="16dp"
+                        android:paddingTop="8dp"
+                        android:paddingBottom="8dp"/>
+
+                </LinearLayout>
+
+                <android.support.v4.widget.NestedScrollView
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:id="@+id/NestedScrollView">
+
+                    <LinearLayout
+                        android:id="@+id/content"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:orientation="vertical"
+                        android:layout_gravity="center_vertical"
+                        android:paddingBottom="8dp"
+                        android:paddingLeft="16dp"
+                        android:paddingRight="16dp"
+                        android:paddingTop="8dp">
+
+                    </LinearLayout>
+
+                </android.support.v4.widget.NestedScrollView>
+
+            </LinearLayout>
+
+        </android.support.design.widget.CoordinatorLayout>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="16dp"
+            android:background="@drawable/more_gradient"
+            android:layout_above="@+id/buttonPanel"
+            />
+
+        <LinearLayout android:id="@+id/buttonPanel"
+                      android:layout_alignParentBottom="true"
+                      android:layout_width="match_parent"
+                      android:layout_height="wrap_content"
+                      android:orientation="horizontal"
+                      android:layoutDirection="locale"
+                      android:background="@android:color/background_light"
+                      style="?android:attr/buttonBarStyle">
+
+            <Button
+                android:id="@+id/buttonOtherAnswers"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight="2"
+                android:text="@string/quest_generic_otherAnswers"
+                style="?android:attr/buttonBarButtonStyle"/>
+
+
+        </LinearLayout>
+    </RelativeLayout>
+
+    <View
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_weight=".5"/>
+
+</de.westnordost.streetcomplete.view.SlidingLinearLayout>
```

Comment:
The elevation-shadows are not drop-shadows. If you look closely, you will see that an elevation-shadow is also visible on the top of the element.